large-message-core/src/main/java/com/bakdata/kafka/AbstractLargeMessageConfig.java

```diff
@@ -73,51 +75,57 @@
  * <ul>
  *     <li> Connection string
  * </ul>
+ * <p></p>
+ * Google Cloud Storage specific
+ * <ul>
+ *     <li> Project ID
```

Comment:
In which case is the project ID sufficient for authentication? I'd have thought you need something like a service account config.

Reply:
Currently, the code supports buckets with public access, but I can dig into the buckets that have private access.